.env.example

```diff
@@ -0,0 +1,10 @@
+PUBLIC_SITE_URL=https://swyxkit.netlify.app
+PUBLIC_APPROVED_POSTERS_GH_USERNAME=sw-yx
+PUBLIC_GH_USER_REPO=sw-yx/swyxkit
+PUBLIC_SITE_TITLE=SwyxKit
+PUBLIC_SITE_DESCRIPTION=swyxs default SvelteKit + Tailwind starter
+PUBLIC_DEFAULT_OG_IMAGE=https://user-images.githubusercontent.com/6764957/147861359-3ad9438f-41d1-47c8-aa05-95c7d18497f0.png
+PUBLIC_MY_TWITTER_HANDLE=swyx
+PUBLIC_MY_YOUTUBE=https://youtube.com/swyxTV
+PUBLIC_POST_CATEGORIES=Blog
+GH_TOKEN=
```

Comment:
i am worried about security - we probably WANT our user to check in all the `PUBLIC_` stuff, but not the `GH_TOKEN`. 

i think these two types of variables should not be in the same file

Reply:
Since the `GH_TOKEN` does not have the prefix `PUBLIC_` it will never be leaked client-side.
If user tries to change `GH_TOKEN` to `PUBLIC_GH_TOKEN` assuming it's a typo, the app will crash on start as all env variables are used must be declared [svelte](https://kit.svelte.dev/docs/modules#$env-static-private)

Reply:
my concern is not about leaking client side, it is for when people push their code to a public github repo. all the PUBLIC_ stuff is ok to leave in the repo, but not the GH_TOKEN stuff.

i'll update readme accordingly

Reply:
All variables are in `.env` file which will never be pushed to git repo.
It is set to be ignored by .gitignore file.